### ISSUE TITLE
link to installation instructions in domoticz forum

### ISSUE THREAD

**Issue by four2six** [allan-gam/GCal#7]
make sure to change it as i moved the installation instructions to a new page

**Comment by four2six** [comment]
the sorting of those pages on the overview on the right is shitty. i added numbers, i know that's not the right way. at the end someone could add a proper index. that would break the link again though.

**Comment by allan-gam** [comment]
Thanks! I don't know how others have solved that but I get Your point. However if we need to reorder the pages or add pages in between them, all external URLs will become invalid. Just a thought. ;)

**Comment by four2six** [comment]
yes, i looked at the issues others have with it... they are all pissed. 
so either you do it with the numbers, or you add an extra sidebar underneath and keep the wrong order on top. Weirdly i don't find any size restrictions to the main wikipage, but last time when i added something it cut off the bottom. But maybe that just had to with the coding. I'm having issues with adding the code of the whole config.json file and adding text after it without either turning the whole thing into one big text chunk or formatting the text after it just as it it was code as well.
Long story short, we could try and name the pages alphabetically without the numbers. Not sure we'd find an appropriate wording though ;-)

**Comment by four2six** [comment]
i changed that. now don't you dare and invent an issue that doesn't fit in there ;)

**Comment by four2six** [comment]
did you change the link in the forum?

**Comment by allan-gam** [comment]
Thanks! I've fixed the link again.
